src/test/java/seedu/organizer/logic/commands/AnswerCommandTest.java

```diff
@@ -0,0 +1,89 @@
+package seedu.organizer.logic.commands;
+
+import static junit.framework.TestCase.assertEquals;
+import static seedu.organizer.testutil.TypicalTasks.ADMIN_USER;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.organizer.logic.CommandHistory;
+import seedu.organizer.logic.UndoRedoStack;
+import seedu.organizer.logic.commands.exceptions.CommandException;
+import seedu.organizer.model.Model;
+import seedu.organizer.model.ModelManager;
+import seedu.organizer.model.user.UserWithQuestionAnswer;
+import seedu.organizer.model.user.exceptions.DuplicateUserException;
+
+//@@author dominickenn
+/**
+ * Contains unit tests for AnswerCommand.
+ */
+public class AnswerCommandTest {
+
+    @Rule
+    public final ExpectedException exception = ExpectedException.none();
+
+    private Model model;
+    private AnswerCommand answerCommand;
+
+    @Before
+    public void setUp() {
+        model = new ModelManager();
+        try {
+            model.addUser(ADMIN_USER);
+        } catch (DuplicateUserException e) {
+            e.printStackTrace();
+        }
+        answerCommand = new AnswerCommand("admin", "answer");
+    }
+
+    @Test
+    public void execute_existingUser_noQuestion() throws Exception {
+        assertCommandSuccess(answerCommand,
+                String.format(AnswerCommand.MESSAGE_NO_QUESTION, "admin"));
+    }
+
+    @Test
+    public void execute_existingUser_answer() throws Exception {
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. name doesn't follow naming convention

Recall that the last world of the method name should be the <expectedBehaviour>